tests/unit/buckets/test_create_bucket.py

```diff
@@ -0,0 +1,27 @@
+"""Unit tests for create bucket."""
+from s3_tools import create_bucket
+
+from tests.unit.conftest import BUCKET_NAME
+
+import pytest
+from botocore.exceptions import ParamValidationError
+
+
+class TestCreate:
+
+    def test_create_bucket(self, s3_client):
+        response = create_bucket(BUCKET_NAME)
+
+        assert response is True
+
+    def test_create_bucket_invalid_name(self, s3_client):
+
+        with pytest.raises(ParamValidationError):
+            create_bucket('')
+
+    def test_create_duplicated_bucket(self, s3_client):
+        create_bucket(BUCKET_NAME)
+
+        response = create_bucket(BUCKET_NAME)
```

Comment:
shouldn't throw an error? Or just ignores since already exist? 🤔

Reply:
So, this is a weird behaviour from boto3. When you create a bucket with the same name, it just returns 200 as the first time. This makes it impossible to know what happened on the AWS side 😞